docs/content/guides/dagster/automated_pipelines.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Automating Your Data Pipelines | Dagster
+description: This guide illustrates how to automate your data pipelines 
+---
+
+# Automating your data pipelines
+
+In this guide, we’ll walk you through how to run your data pipelines without manual intervention, i.e. automate, and identify the Dagster tools to make that happen.
+
+This guide assumes you have some familiarity with several Dagster concepts, including [software-defined assets,](https://docs.dagster.io/concepts/assets/software-defined-assets) [ops](https://docs.dagster.io/concepts/ops-jobs-graphs/ops) and [graphs](https://docs.dagster.io/concepts/ops-jobs-graphs/graphs), and [jobs](https://docs.dagster.io/concepts/ops-jobs-graphs/jobs), and [when to use each](https://docs.dagster.io/guides/dagster/how-assets-relate-to-ops-and-graphs).
```

Comment:
I don't think users actually need to understand ops or graphs in order to use this guide. What do you think

Reply:
Updated